### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by jordijoangimenez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,11 +84,49 @@ specify an alternative configuration file. For example:
...
+[Testing the M1 Interface on v1.3.0](https://5g-mag.github.io/Getting-Started/pages/5g-media-streaming/rt-5gms-application-function/testing-m1-v130.html) page.
+
+For testing the M1 interface on 5GMSd Application Function v1.4.1 or later, then please visit the
+[Testing the M1 Interface on v1.4.1](https://5g-mag.github.io/Getting-Started/pages/5g-media-streaming/rt-5gms-application-function/testing-m1-v141.html) page.
+
+### Testing the M3 Interface
+
+Depending on which version of the 5GMSd Application Function you wish to test, the commands to test the interface at reference point M3 change.
+
+If you wish to test 5GMSd Application Function v1.1.x then please see the [Testing the M3 Interface on v1.1.0](https://5g-mag.github.io/Getting-Started/pages/5g-media-streaming/rt-5gms-application-function/testing-m3-v110.html) page.
```
Comment (dsilhavy) [inline README.md:110]: Broken link, needs to be added to getting started

File: README.md [hunk 2]
```diff
@@ -84,11 +84,49 @@ specify an alternative configuration file. For example:
...
+
+For versions after v1.1.x (i.e. v1.2.0 and above) please use the [Testing the M3 Interface on v1.2.0](https://5g-mag.github.io/Getting-Started/pages/5g-media-streaming/rt-5gms-application-function/testing-m3-v120.html) page.
```
Comment (dsilhavy) [inline README.md:112]: Broken link, needs to be added to getting started

File: README.md [hunk 3]
```diff
@@ -84,11 +84,49 @@ specify an alternative configuration file. For example:
...
+
+### Testing: M5 Interface
+
+The details of these tests change with different versions of the 5GMSd Application Function.
+
+If you are testing versions up to v1.1.x then please visit the [Testing: M5 Interface on v1.0.0](https://5g-mag.github.io/Getting-Started/pages/5g-media-streaming/rt-5gms-application-function/testing-m5-v100.html)
```
Comment (dsilhavy) [inline README.md:118]: Broken link, needs to be added to getting started

File: README.md [hunk 4]
```diff
@@ -84,11 +84,49 @@ specify an alternative configuration file. For example:
...
+page.
+
+If you are testing the M5 interface on 5GMSd Application Function v1.2.x please visit the
+[Testing the M5 Interface on v1.2.0](https://5g-mag.github.io/Getting-Started/pages/5g-media-streaming/rt-5gms-application-function/testing-m5-v120.html) page.
```
Comment (dsilhavy) [inline README.md:122]: Broken link, needs to be added to getting started

File: README.md [hunk 5]
```diff
@@ -84,11 +84,49 @@ specify an alternative configuration file. For example:
...
+
+If you are testing the M5 interface on 5GMSd Application Function v1.3.0 or later please visit the
+[Testing the M5 Interface on v1.3.0](https://5g-mag.github.io/Getting-Started/pages/5g-media-streaming/rt-5gms-application-function/testing-m5-v130.html) page.
```
Comment (dsilhavy) [inline README.md:125]: Broken link, needs to be added to getting started

File: README.md [hunk 6]
```diff
@@ -84,11 +84,49 @@ specify an alternative configuration file. For example:
...
+
+### Testing with Postman
+
+For detailed instructions on how to use the Postman Collection please refer to this [documentation](https://5g-mag.github.io/Getting-Started/pages/5g-media-streaming/rt-5gms-application-function/testing-postman.html).
```
Comment (dsilhavy) [inline README.md:129]: Broken link, needs to be added to getting started

**Comment by dsilhavy** [comment]
@jordijoangimenez Can you please fix the broken links now that we merged https://github.com/5G-MAG/Getting-Started/pull/65

**Review by dsilhavy — CHANGES_REQUESTED** [review]

**Review by jordijoangimenez — COMMENTED** [review]
All fixed

**Review by dsilhavy — APPROVED** [review]

**Pull request merged**